attributes.rst

```diff
@@ -0,0 +1,146 @@
+.. index::
+    single: Attributes
+
+Symfony Attributes Overview
+===========================
+
+Attributes are the successor of annotation since PHP 8. Attributes are native
+to the language and Symfony takes full advantage of them across the framework
+ans its different components.
+
+This article lists supported attributes of the framework, as well as their
+usages in the documentation.
+
+Doctrine Bridge
+~~~~~~~~~~~~~~~
+
+* :doc:`UniqueEntity </reference/constraints/UniqueEntity>`
+
+Command
+~~~~~~~
+
+* :ref:`AsCommand <console_registering-the-command>`
+
+Contracts
+~~~~~~~~~
+
+* :ref:`Required <autowiring-calls>`
+* :ref:`SubscribedService <service-subscribers-service-subscriber-trait>`
+
+Dependency Injection
+~~~~~~~~~~~~~~~~~~~~
+
+* :ref:`AsTaggedItem <tags_as-tagged-item>`
+* :ref:`Autoconfigure <lazy-services_configuration>`
+* :ref:`AutoconfigureTag <di-instanceof>`
+* :ref:`TaggedIterator <tags_reference-tagged-services>`
+* :ref:`TaggedLocator <service-subscribers-locators_defining-service-locator>`
+* :ref:`Target <autowiring-multiple-implementations-same-type>`
+* :ref:`When <service-container_limiting-to-env>`
+
+EventDispatcher
+~~~~~~~~~~~~~~~
+
+* :ref:`AsEventListener <event-dispatcher_event-listener-attributes>`
+
+HttpKernel
+~~~~~~~~~~
+
+* :doc:`AsController </controller/service>`
+
+Messenger
+~~~~~~~~~
+
+* :ref:`AsMessageHandler <messenger-handler>`
+
+Routing
+~~~~~~~
+
+* :doc:`Route </routing>`
+
+Security
+~~~~~~~~
+
+* :class:`Symfony\\Component\\Security\\Core\\Validator\\Constraints\\UserPassword`
+* :ref:`CurrentUser <security-json-login>`
+
+Serializer
+~~~~~~~~~~
+
+* :ref:`Context <serializer_serializer-context>`
+* :ref:`DiscriminatorMap <serializer_interfaces-and-abstract-classes>`
+* :ref:`Groups <component-serializer-attributes-groups-annotations>`
+* :ref:`Ignore <serializer_ignoring-attributes>`
+* :ref:`MaxDepth <serializer_handling-serialization-depth>`
+* :ref:`SerializedName <serializer_name-conversion>`
+
+Validator
+~~~~~~~~~
+
+* :doc:`All </reference/constraints/All>`
```

Comment:
This list will be hard to maintain, whenever a new constraint is introduced we now must update 2 lists instead of one. As the [other list](https://github.com/symfony/symfony-docs/blob/6.2/reference/constraints/map.rst.inc) doesn't use a format suited for this document, I suggest replacing the list of constraints with:

```rst
Each validation constraint comes with a PHP attribute. See :doc:`/reference/constraints` for
a full list of validation constraints.
```